src/JuliusSweetland.OptiKey.Core/UI/ViewModels/MainViewModel.ServiceEventHandlers.cs

```diff
@@ -128,6 +124,15 @@ private void SetupInputServiceEventHandlers()
                 if (SelectionMode == SelectionModes.Key
                     && (singleKeyValue != null || (multiKeySelection != null && multiKeySelection.Any())))
                 {
+                    if (singleKeyValue != null && singleKeyValue.FunctionKey != null && singleKeyValue.FunctionKey.Value == FunctionKeys.StepList)
+                    {
+                        await StepList(singleKeyValue, multiKeySelection);
+
+                        //do stuff on another thread 

```

Comment:
Minor cleanup - could you remove these commented out lines pls.

Reply:
It is done.